apps/communications/dialer/test/unit/call_log_test.js

```diff
@@ -29,6 +29,9 @@ require('/shared/test/unit/mocks/mock_image.js');
 require('/shared/test/unit/mocks/mock_sim_settings_helper.js');
 require('/shared/test/unit/mocks/dialer/mock_contacts.js');
 require('/shared/test/unit/mocks/mock_mozContacts.js');
+require('/shared/js/fb/fb_request.js');
+require('/contacts/test/unit/mock_fb.js');
```

Comment:
Rather than adding more dependencies to the Contacts app, we should refactor this and request review from a Contacts peer.